strawberryfields/__init__.py

```diff
@@ -32,6 +32,7 @@
 from .tdm import TDMProgram
 from .plot import plot_wigner, plot_fock, plot_quad
 from . import tdm
+from .backends.tfbackend.ops import update_symplectic
```

Comment:
It will cause a failure in that case indeed. If `update_symplectic` will only be use within the context of having TensorFlow installed, perhaps this import can be moved to [`strawberryfields/backends/tfbackend/__init__.py`](https://github.com/sylviemonet/strawberryfields/blob/6cabcab5143007643489d2bb49ea8207be7caa4e/strawberryfields/backends/tfbackend/__init__.py#L144-L151) after checking if `tf` is installed.

Reply:
I agree with @sduquemesa that this would better be moved to the `tfbackend` init file instead, if it needs to live withing the `tfbackend.ops.py` file. Alternatively just keep it where it is (and just remove it from any init files), as `strawberryfields.backends.tfbackend.ops.update_symplectic`. 

Although, I'm wondering why this function is inside `tfbackend` at all. Since it doesn't seem to use TensorFlow, wouldn't it be better just to move it outside of the `tfbackend` files? Would this function perhaps even make sense to put in The Walrus, in the `thewalrus.symplectic` submodule? @ziofil ? 🤔

Reply:
> Since it doesn't seem to use TensorFlow, wouldn't it be better just to move it outside of the tfbackend files? 

Agree!

Reply:
Just checking if anyone is available to work on this? Just to avoid this PR being blocked for too much longer 🙂